FAIL
Kernel panic in run: PANIC at ../../threads/synch.c:248 in lock_release(): assertion `lock_held_by_current_thread (lock)' failed.
Call stack: 0x8004213c38 0x800420a7c6 0x8004217e26 0x800421789e 0x800421979c 0x80042190e4 0x8004218efbc
Translation of call stack:
0x0000008004213c38: debug_panic (lib/kernel/debug.c:32)
0x000000800420a7c6: lock_release (threads/synch.c:250)
0x0000008004217e26: load (userprog/process.c:487)
0x000000800421789e: process_exec (userprog/process.c:259)
0x000000800421979c: exec (userprog/syscall.c:257)
0x00000080042190e4: syscall_handler (userprog/syscall.c:93)
0x000008004218efbc: (unknown)
